hackle-android-sdk/src/main/java/io/hackle/android/internal/pii/PIIEventManager.kt

```diff
@@ -0,0 +1,47 @@
+package io.hackle.android.internal.pii
+
+import io.hackle.android.internal.pii.phonenumber.PhoneNumber
+import io.hackle.android.internal.user.UserManager
+import io.hackle.sdk.common.Event
+import io.hackle.sdk.common.PropertyOperations
+import io.hackle.sdk.common.User
+import io.hackle.sdk.core.HackleCore
+
+internal class PIIEventManager(
+    private val userManager: UserManager,
+    private val core: HackleCore,
+) {
+    fun setPhoneNumber(phoneNumber: String, user: User, timestamp: Long) {
```

Comment:
```suggestion
    fun setPhoneNumber(phoneNumber: String, timestamp: Long) {
    
    val hackleUser = userManager.resolve(null)
```
User를 파라미터로 받지 않고 내부적으로 처리하는게 좋을듯합니다